### PR TITLE
added sheet component

### PR DESCRIPTION
**Pull request by ChiSamurai**

### CODE REVIEW

File: packages/pds-components/src/sheet/sheet.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  position: relative;
+
+  @include pds.inner-spacing($sheet-padding-x, x);
+  @include pds.inner-spacing($sheet-padding-y, bottom);
+
+  &:first-child {
+    @include pds.inner-spacing($sheet-padding-y, top);
+  }
+
+  > ul {
```
Comment (JanUnld) [inline packages/pds-components/src/sheet/sheet.scss:25]: Why is there a styling specifically aimed for `ul` elements inside the potential `pds-sheet` scope?

**Review by JanUnld — CHANGES_REQUESTED** [review]
Thanks for contributing ❤️! Unfortunately there's already a kind of sheet "component" available 🙈 using...

```scss
@use "styleguide" as pds;

@include pds.sheet;
```
```html
<div class="sheet"></div>
```

It would generally be nice to know some more information behind the purpose of the pull request. Some parts seem unclear at the moment. I added a comment 😉

**Comment by JanUnld** [comment]
Also as a side note when committing, please respect the https://www.conventionalcommits.org/en/v1.0.0/ spec. We are looking forward to provide some contribution guidelines soon, also touching up on the commit convention topic 😉

**Comment by maximni-vg** [comment]
> It would generally be nice to know some more information behind the purpose of the pull request. Some parts seem unclear at the moment.

The reason was to propose an implementation of the sheet component as an actual component, instead of "a kind of 'component'". How is it planned to handle changes like this? Moving such components into a "project"-specific component kit parallel to pds-components?

**Comment by JanUnld** [comment]
Sounds good 👍. I think a separate component impl for the `pds-sheet` case is fine.

> How is it planned to handle changes like this?

Unfortunately, it isn't. There's no clear contribution guidance as of now. **But** I think we should be able to merge such things anyways, **if** they are discussed and reviewed properly. So for our current situation I would like to have...

1. a commit history, that respects the "conventional commit" spec (to be not loose the ability for tooling related to changelogs and the git history generally)
2. the `scss` related sheet code removed or moved to the component, that currently implements the `.sheet` class and it's styling
3. a deeper understanding of the `ul` handling (see https://github.com/vitagroupag/pds/pull/2#discussion_r703067366)

We can schedule a call if you'd like 🙂

**Pull request closed without merging**